lld/MachO/SymbolTable.cpp

```diff
@@ -204,8 +204,9 @@ Defined *SymbolTable::addDefined(StringRef name, InputFile *file,
 
   // With -flat_namespace, all extern symbols in dylibs are interposable.
   // FIXME: Add support for `-interposable` (PR53680).
```

Comment:
```suggestion
```
This can be removed, no?

Reply:
Duly removed in a --amend push along with the `Author:` commit header. I'm completely unfamiliar with how you would write such a test for this change inside this project. Could you point me to any resources that might be helpful or a similar test please?

Reply:
Have a look at https://github.com/llvm/llvm-project/tree/main/lld/test. In particular, the MachO directory.

These links might also help:
- https://www.llvm.org/docs/TestingGuide.html
- https://llvm.org/docs/CommandGuide/lit.html
- https://llvm.org/docs/CommandGuide/FileCheck.html

Reply:
Thanks for the info @carlocab, I was able to adapt an existing test to test symbols interpose with the new flag.